### PR TITLE
docs: add VMware Getting Started DOC-1126

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This PR makes the following changes: 

- Add the VMware section, including PCG deployment. 
- Updated all cluster profile creation flows across clouds to set the 3-tier app preset. 
- Updated all Deploy Cluster Profile Updates pages to add the Kubecost pack and explore its UI. 
- Added a section on Cluster Observability to all Cluster Profile Updates pages.

## Changed Pages

💻 [Getting Started](https://deploy-preview-3173--docs-spectrocloud.netlify.app/getting-started/)

## Jira Tickets

🎫 [DOC-1126](https://spectrocloud.atlassian.net/browse/DOC-1126)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._
- [x] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1126]: https://spectrocloud.atlassian.net/browse/DOC-1126?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/getting-started/aws/update-k8s-cluster.md [hunk 1]
```diff
@@ -220,21 +227,24 @@ tag. Palette indicates that the cluster associated with the cluster profile you
...
+![Image that shows the Updates button](/getting-started/aws/getting-started_update-k8s-cluster_updates-available-button-cluster-overview.webp)
 
 A dialog appears which shows the changes made in this update. Review the changes and ensure the only change is the
 `replicas` field value. The pending update removes your cluster override and sets the `replicas` field to `3`. At this
 point, you can choose to apply the pending changes or keep it by modifying the right-hand side of the dialog.
 
 ![Image that shows the available updates dialog ](/getting-started/aws/getting-started_update-k8s-cluster_available-updates-dialog.webp)
 
-Click on **Confirm updates** once you have finished reviewing your changes.
+Click on \*Apply Changes\*\* once you have finished reviewing your changes.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/getting-started/aws/update-k8s-cluster.md:238]: I noticed that for the azure tutorial we have one more sentence after: This removes your cluster override.

File: docs/docs-content/getting-started/vmware/vmware.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: "Deploy a Cluster to VMware"
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/vmware/vmware.md:2]: The VMware card is not present on the getting started index page :)

File: docs/docs-content/getting-started/vmware/vmware.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+title: "Deploy a Cluster to VMware"
+description: "Spectro Cloud Getting Started with VMware"
+hide_table_of_contents: false
+sidebar_custom_props:
+  icon: ""
+tags: ["getting-started", "vmware"]
+---
+
+Palette supports using [VMware](https://www.vmware.com) as a data center provider. You can deploy and manage
+[Host Clusters](../../glossary-all.md#host-cluster) on VMware virtual machines.
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/vmware/vmware.md:12]:
```suggestion
Palette supports integration with [VMware](https://www.vmware.com). You can deploy and manage
[Host Clusters](../../glossary-all.md#host-cluster) on VMware.
```
I think I would follow the same pattern we used for the other providers, but its up to you :)

File: docs/docs-content/getting-started/vmware/setup.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+3. Navigate to the left **Main Menu** and select **Tenant Settings**.
+
+4. From the **Tenant Settings Menu**, select **API Keys**.
+
+5. Verify the API key is listed in the table with the correct user name and expiration date.
+
+## Next Steps
+
+Now that you set up Palette for use with VMware vSphere, you can start deploying Kubernetes clusters to your VMware user
+account.
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/vmware/setup.md:51]: This is not entirely true :( You can only deploy clusters to vmware if you have an active PCG. When you deploy a PCG, a vmware cloud account is automatically created

File: docs/docs-content/getting-started/vmware/update-k8s-cluster.md [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+
+Palette creates a new cluster profile version and opens it. The version dropdown displays the newly created **1.1.0**
+profile. This profile version is not deployed to any host clusters.
+
+![Image that shows cluster profile version 1.1.0](/getting-started/vmware/getting-started_update-k8s-cluster_new-version-overview.webp)
+
+The version **1.1.0** has the same layers as the version **1.0.0** it was created from.
+
+Click on **Add New Pack**. Select the **Public Repo** registry and scroll down to the **Monitoring** section. Find the
+**Kubecost** pack and select it. Alternatively, you can user the search function with the pack name **Kubecost**.
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/vmware/update-k8s-cluster.md:105]:
```suggestion
**Kubecost** pack and select it. Alternatively, you can use the search function with the pack name **Kubecost**.
```

File: docs/docs-content/getting-started/vmware/update-k8s-cluster.md [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+will be successfully deployed.
+
+![Image that shows completed cluster profile updates](/getting-started/vmware/getting-started_update-k8s-cluster_completed-cluster-updates.webp)
+
+Download the [kubeconfig](../../clusters/cluster-management/kubeconfig.md) file for your cluster from the Palette UI.
+This file enables you and other users to issue kubectl commands against the host cluster.
+
+![Image that the kubeconfig file](/getting-started/vmware/getting-started_update-k8s-cluster_download-kubeconfig.webp)
+
+Open a terminal window and set the environment variable `KUBECONFIG` to point to kubeconfig file you downloaded.
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/vmware/update-k8s-cluster.md:146]:
```suggestion
Open a terminal window and set the environment variable `KUBECONFIG` to point to the kubeconfig file you downloaded.
```

File: docs/docs-content/getting-started/vmware/update-k8s-cluster.md [hunk 7]
```diff
@@ -0,0 +1,292 @@
...
+
+```shell
+export KUBECONFIG=~/Downloads/admin.vmware-cluster.kubeconfig
+```
+
+Forward the Kubecost UI to your local network. The Kubecost dashboard is not exposed by default, so the command below
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/vmware/update-k8s-cluster.md:152]:
```suggestion
Forward the Kubecost UI to your local network. The Kubecost dashboard is not exposed externally by default, so the command below
```

File: docs/docs-content/getting-started/vmware/update-k8s-cluster.md [hunk 8]
```diff
@@ -0,0 +1,292 @@
...
+will allow you to access it locally.
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/vmware/update-k8s-cluster.md:153]:
```suggestion
will allow you to access it locally on port **9090**. If port 9090 is already taken, you can choose a different one.
```
maybe?

File: docs/docs-content/getting-started/vmware/update-k8s-cluster.md [hunk 9]
```diff
@@ -0,0 +1,292 @@
...
+
+Once the cluster is deleted, navigate to the left **Main Menu** and click on **Profiles**. Find the cluster profile you
+created and click on the **three-dot Menu** to display the **Delete** button. Select **Delete** and confirm the
+selection to remove the cluster profile.
+
+<PartialsComponent category="pcg-vmware" name="delete-pcg-ui" />
+
+## Wrap-Up
+
+In this tutorial, you created deployed cluster profile updates. After the cluster was deployed to AWS, you updated the
```
Comment (caroldelwing) [inline docs/docs-content/getting-started/vmware/update-k8s-cluster.md:285]:
```suggestion
In this tutorial, you created deployed cluster profile updates. After the cluster was deployed to VMware, you updated the
```

**Review by caroldelwing — APPROVED** [review]
Awesome work! :clap:
I left a few tiny comments :)

File: docs/docs-content/getting-started/aws/update-k8s-cluster.md [hunk 10]
```diff
@@ -143,15 +133,38 @@ backups before you make any cluster profile version changes in your production e
...
+![Image that the kubeconfig file](/getting-started/aws/getting-started_update-k8s-cluster_download-kubeconfig.webp)
+
+Open a terminal window and set the environment variable `KUBECONFIG` to point to kubeconfig file you downloaded.
+
+```shell
+export KUBECONFIG=~/Downloads/admin.aws-cluster.kubeconfig
+```
+
+Forward the Kubecost UI to your local network. The Kubecost dashboard is not exposed by default, so the command below
+will allow you to access it locally.
```
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/getting-started/aws/update-k8s-cluster.md:153]: Excellent shout!

File: docs/docs-content/getting-started/vmware/setup.md (hunk 4 above)
Comment (addetz) [reply docs/docs-content/getting-started/vmware/setup.md]: Excellent point. I have edited this section to reference the PCG instead.

File: docs/docs-content/getting-started/vmware/vmware.md (hunk 2 above)
Comment (addetz) [reply docs/docs-content/getting-started/vmware/vmware.md:2]: Thanks for spotting that! Added!

**Pull request merged**